src/component/button/HsiButton/HsiButton.tsx

```diff
@@ -181,17 +200,35 @@ export class HsiButton extends React.Component<HsiButtonProps, HsiButtonStatePro
             }
           );
 
-      featureInfoUrls.push(featureInfoUrl);
-
+      if (featureInfoUrl.indexOf('geoserver.action') >= 0) {
+        const ftName: string = layer.getSource().getParams().LAYERS;
+        const namespace: string = ftName.indexOf(':') > -1 ? ftName.split(':')[0] : ftName;
```

Comment:
Do we have a util for this somewhere? If not, should we probably have one?

Reply:
not yet, great idea for another PR